tests/Microsoft.Bot.Builder.Core.Extensions.Tests/TranscriptUtilities.cs

```diff
@@ -23,18 +23,62 @@ public static class TranscriptUtilities
         /// <param name="context">Test context</param>
         /// <returns>A list of activities to test</returns>
         public static IEnumerable<IActivity> GetFromTestContext(TestContext context)
+        {
+            var relativePath = Path.Combine(context.FullyQualifiedTestClassName.Split('.').Last(), $"{context.TestName}.chat");
+            return GetActivities(relativePath);
+        }
+
+        public static IEnumerable<IActivity> GetActivities(string relativePath)
         {
             var transcriptsRootFolder = TestUtilities.GetKey("TranscriptsRootFolder") ?? @"..\..\..\..\..\transcripts";
-            var directory = Path.Combine(transcriptsRootFolder, context.FullyQualifiedTestClassName.Split('.').Last());
-            var fileName = $"{context.TestName}.transcript";
-            var path = Path.Combine(directory, fileName);
+            var path = Path.Combine(transcriptsRootFolder, relativePath);
+            if (!File.Exists(path))
+            {
+                path = Path.Combine(transcriptsRootFolder, relativePath.Replace(".chat", ".transcript", StringComparison.InvariantCultureIgnoreCase));
+            }
             if (!File.Exists(path))
             {
                 Assert.Fail($"Required transcript file '{path}' does not exists in '{transcriptsRootFolder}' folder. Review the 'TranscriptsRootFolder' environment variable value.");
             }
 
-            var content = File.ReadAllText(path);
-            return JsonConvert.DeserializeObject<Activity[]>(content);
+            string content;
+            if (string.Equals(path.Split('.').Last(), "chat", StringComparison.InvariantCultureIgnoreCase))
+            {
+                content = Chatdown(path);
+            }
+            else
+            {
+                content = File.ReadAllText(path);
+            }
+
+            var activities = JsonConvert.DeserializeObject<List<Activity>>(content);
+
+            var lastActivity = activities.Last();
+            if (lastActivity.Text.Last() == '\n')
+            {
+                lastActivity.Text = lastActivity.Text.Remove(lastActivity.Text.Length - 1);
+            }
+
+            return activities.Take(activities.Count - 1).Append(lastActivity);
+        }
+
+        private static string Chatdown(string path)

```

Comment:
Check to see if chatdown is installed.
Start Process.Start will throw the following exception if chatdown is not installed:

`System.ComponentModel.Win32Exception: The system cannot find the file specified`

Reply:
Now chatdown is called from a `.cmd` file. This prevents unwanted execution exceptions.
If chatdown conversion fails, the corresponding exception will be raised.